pyc_module.cpp

```diff
@@ -227,3 +244,51 @@ PycRef<PycObject> PycModule::getRef(int ref) const
         throw std::out_of_range("Ref index out of range");
     return *it;
 }
+
+#define enumVariant(mj, mn) case mn: \
+    return PycMagic::MAGIC_ ## mj ## _ ## mn
+
+PycMagic version_to_magic(int major, int minor) {
+    switch (major) {
+        case 1:
+        switch (minor) {
+            enumVariant(1, 0);
+            case 1:
+            case 2:
+            return PycMagic::MAGIC_1_1;
+            enumVariant(1, 3);
+            enumVariant(1, 4);
+            enumVariant(1, 5);
+            enumVariant(1, 6);
+            default: return PycMagic::INVALID;
+        }
+        case 2:
+            switch (minor) {
+            enumVariant(2, 0);
+            enumVariant(2, 1);
+            enumVariant(2, 3);
+            enumVariant(2, 4);
+            enumVariant(2, 5);
+            enumVariant(2, 6);
+            enumVariant(2, 7);
+            default: return PycMagic::INVALID;
+        }
+        case 3:
+            switch (minor) {
+            enumVariant(3, 0);
+            enumVariant(3, 1);
+            enumVariant(3, 3);
+            enumVariant(3, 4);
+            enumVariant(3, 5);
+            enumVariant(3, 6);
+            enumVariant(3, 7);
+            enumVariant(3, 8);
+            enumVariant(3, 9);
+            enumVariant(3, 10);
+            default: return PycMagic::INVALID;
+        }
+        default:
+            return PycMagic::INVALID;
+        }
+    }
```

Comment:
```suggestion
    }
}
```

The indentation is a bit inconsistent in this switch, which I think caused the ending braces to be misaligned...
Generally, the pycdc style is to keep `case` aligned with the `switch`:
```c++
switch (<cond>) {
case <N>:
    <stuff>
    break;
case <M>:
    {
        <stuff>
    }
    break;
default:
    <stuff>
    break;
}
```